implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -25,6 +25,7 @@
 import io.micrometer.core.instrument.util.HierarchicalNameMapper;
 import io.micrometer.core.lang.Nullable;
 import io.micrometer.statsd.internal.*;
+import org.apache.commons.lang.ArrayUtils;
```

Comment:
We don't want to make a hard dependency on Apache Commons, but I can fix this while merging.